src/nvim/keycodes.c

```diff
@@ -572,7 +572,7 @@ char *get_special_key_name(int c, int modifiers)
 /// @param[out]  did_simplify  found <C-H>, etc.
 ///
 /// @return Number of characters added to dst, zero for no match.
-unsigned int trans_special(const char **const srcp, const size_t src_len, char *const dst,
+unsigned     trans_special(const char **const srcp, const size_t src_len, char *const dst,
```

Comment:
hmm, is it a reason this becomes empty space instead of deleting it? (not a blocker, but seems curios)

Reply:
Just a bug. Manually fixed, and reformatting doesn't reintroduce it.